osidb/models/flaw/label.py

```diff
@@ -76,19 +167,40 @@ class FlawCollaboratorState(models.TextChoices):
 
     contributor = models.CharField(max_length=255, blank=True)
 
+    relevant = models.BooleanField(default=True)
+
+    objects = FlawCollaboratorManager.from_queryset(CustomQuerySetUpdatedDt)()
+
     class Meta:
         constraints = [
             models.UniqueConstraint(
                 fields=["flaw", "label"], name="unique label per flaw"
             ),
         ]
 
+    def create(self, *args, **kwargs):
+        self._validate_label()
+        super().create(*args, **kwargs)
```

Comment:
I assume that this is for some case when we bypass `save` which should better not happen :smile: What is the reason for it? Also explicitly calling validations on various places or raising `ValidationError` directly from `save` seems all together already a bit hacky to me. Could we use `AlertMixin` instead?

Reply:
OK :thinking: I see! So the source of the weirdness is that the creation is more strict than the update. So probably leave it as it is and we shall see.